scripts/superdesk-authoring/metadata/metadata.js

```diff
@@ -252,8 +252,14 @@ function MetaDropdownDirective($filter, keyboardManager) {
             scope.select = function(item) {
                 var o = {};
 
-                if (angular.isDefined(item)) {
-                    o[scope.field] = (scope.field === 'place' || scope.field === 'genre') ? [item] : item.value;
+                if (item) {
+                    if (scope.field === 'place' || scope.field === 'genre') {
```

Comment:
imo it would be better to have some param for this on directive scope, and then it should use qcode if item has one , if not then value.
so avoiding to hardcode some field names